doc/procedures/jira_workflow.rst

```diff
@@ -0,0 +1,86 @@
+Jira workflow
+=============
+
+The Sprint in which these tasks are placed is defined :ref:`here <procedures/jira:Sprints>`.
+
+Please refer to the TSSW JIRA Workflow diagram above.
+
+Initial/Triage
+--------------
+
+* Create a Task, Bug or improvement Jira ticket.
+* The initial assignee should triage the ticket.
+
+  * Ensure it is assigned correctly (Assignee, Component , etc)
+
+    * Ensure the ticket is assigned to an actual, currently employed at LSST, person.
+    * Each ticket is assigned a JIRA Epic.
+    * A label with the name of the component should be added to the ticket.
+    * Each ticket is assigned a component that begins with ``ts_``
+
+
+  * Ensure the Priority is set correctly (Ticket should not have the "Undefined" priority).
+  * A Sprint can be chosen at creation (preferably by the person doing the work for this ticket).
+    Otherwise the ticket is automatically placed in the Backlog.
+    If the ticket is in Backlog, once the sprint is known, the ticket should be updated with the current sprint.
+  * The urgent label is reserved for work that impacts nighttime activities at the summit and should be given the utmost priority.
+
+* Once work begins, move the ticket to In Progress
+
+In Progress
+-----------
+
+* Create the ticket branch in the git repos
+  The branch should be named ``tickets/DM-12345``, where ``DM-12345`` is the ticket number assigned by Jira.
+  Once the ticket is pushed to GitHub it is automatically linked to the JIRA component ticket.
+
+  * No active development is ever done on the Main or Develop branches.
+
+* Write the code.
+* Write the unit tests.
+* Update documentation
+* When complete, move to In Review.
+
+  * Complete meaning:
+
+    * Add a link of the PR to the ticket.
+    * Unit tests exist, have been successfully run and results have been added to the ticket or PR.
+    * Add a link to the generated documentation site
+
+In Review
+---------
+
+* Once the code is complete and all unit tests are passing, initiate a pull request on the develop branch and assign it to the Reviewer(s).
+* The Reviewer ensures
+
+  * Code is complete and understandable.
+  * CI is passing.
+  * Documentation is done, including a reference to the lsst.io site
+  * All requirements, as defined in the ticket description, are met.
+
+* If issues are found
+
+  * Update Jira with issue
+
+    * Comments on GitHub PR are considered acceptable as well
+
+  * Sends back to developer
+
```

Comment:
We may want to add a phrase or two about conflict resolution, when the developer and the reviewer don't see eye to eye... Maybe add:

> * If conflicts arise during the review process, developers should reach out to the Scrum Master and/or the Software Architect to seek a conflict resolution.
>  If a resolution can't be found the issue can be escalated further to the Product Owner and the  Software Manager.

Reply:
Done